### ISSUE TITLE
Right axis deviation

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6607]
**Preferred term label:** Right axis deviation

**Synonyms**
RAD

**Definition (free text, please give PubMed ID)**
Right axis deviation (RAD) involves the direction of depolarisation being distorted to the right (between +90º and +180º).

**Parent term (use hpo.jax.org/app)**
Abnormality of heart axis (once available)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
